### PR TITLE
Created initial readme content for root folder

### PR DESCRIPTION
**Pull request by llunn**
Reference ISSUE #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
-# SEProject
-Software Engineering Course Group Project
+# CSCI 3130 - Software Engineering, Winter 2018
+This project is a group project for the Winter 2018 offering of Software Engineering.
+
+# Overview
+
+The project will consist of the development of an athlete monitor application that allows users to develop self-administered, and coach-driven training routines.  The application is primarily am exercise routine monitor and communication tool.
```
Comment (nickburris) [inline README.md]: "primarily *an* exercise"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,88 @@
...
+# Stable Features
+
+No iterations have been released.
+
+# Iteration 0
+
+Iteration 0 includes the software engineering activities required to seed the agile development cycle.  It includes:
+
+1. Setting up the Github environment;
+1. Itegrating Waffle;
```
Comment (nickburris) [inline README.md]: Integrating

**Comment by llunn** [comment]
Completed #1

**Comment by llunn** [comment]
fixes #1

**Pull request merged**